app/gui/view/graph-editor/src/builtin/visualization/java_script/scatterPlot.js

```diff
@@ -49,7 +49,9 @@ class ScatterPlot extends Visualization {
 
     constructor(data) {
         super(data)
-        this.setPreprocessor('process_to_json_text', 'Standard.Visualization.Scatter_Plot')
+        this.bounds = null
+        this.limit = 32
```

Comment:
Please add it to the constants list on the top of the file.

Reply:
Done in [6dbd742](https://github.com/enso-org/enso/pull/3655/commits/6dbd74202f099e8e17491f430d56b55c22888f9e)